contentcuration/contentcuration/frontend/shared/views/ResizableNavigationDrawer.vue

```diff
@@ -96,9 +96,11 @@
       this.throttledUpdateWidth = animationThrottle((...args) => updateWidth(...args));
 
       this.$nextTick(() => {
-        const drawerBorder = this.drawerElement.querySelector('.v-navigation-drawer__border');
-        drawerBorder.addEventListener('mousedown', this.handleMouseDown, false);
-        document.addEventListener('mouseup', this.handleMouseUp, false);
+        if (this.drawerElement) {
```

Comment:
Maybe this whole `this.$nextTick` block should just turn into a watcher on the `drawerElement`?

Reply:
Yeah, that would work, feels like it should really be in the `mounted` method of the component it is changing (but we can't because it's vuetify).